### ISSUE TITLE
Trying to get downloads folder working on unraid

### ISSUE THREAD

**Issue by elliottmarter** [TannerNelson16/playlistdl#6]
I have the compose setup as per your code and also an env file with the following.

AUDIO_DOWNLOAD_PATH=/mnt/user/media/playlistdl

I find this a bit odd but I guess theres nothing technically wrong with it.

However when I spin up the container files keep getting put in /app/downloads/GUID_STRING

I can cd in to /mnt/user/media/playlistdl just fine from within the container, so it is mounted OK.

Any ideas?

**Comment by TannerNelson16** [comment]
Yeah, the purpose of the .env file is to eliminate having to copy the path in the docker compose file so many times because the folder path is used both in the python script as well as to expose that folder path from the container to the actual server. 

Do you have the proper permissions set in that folder?

**Comment by elliottmarter** [comment]
Yeah permissions seem fine, I was able to `touch test` within the folder to create a file from within the docker CLI.

No matter what I try it just keeps downloading stuff to /app/downloads/GUID_STRING

Are there any logs I can send you to help?

**Comment by TannerNelson16** [comment]
Interesting. Are you logged in as admin while this is happening? The unique folder should only be generated when the user is not logged in. This allows for multiple users downloading at the same time, and is subsequently deleted after the client receives the file(s). Maybe I need to look at the fallback case. Perhaps it's going into a weird state. If you could send the logs for this container, that would be helpful.

**Comment by elliottmarter** [comment]
Hi sure,

it looks like debug is off, let me know how to enable and then I can give it another go and send it over.

Sorry for the delayed reply :)

**Comment by TannerNelson16** [comment]
Just run 'docker logs playlistdl' and it should show you the logs.

**Comment by TannerNelson16** [comment]
Added some additional debugging messages. Let me know if this helps. Going to close for now.
